src/name_bazaar/ui/pages/manage_names_page.cljs

```diff
@@ -0,0 +1,358 @@
+            (when correct-subname?
+              (str
+               full-subname " will be created"))]]]
+         [ui/GridRow
+          {:centered true}
+          [:div
+           [transaction-button
+            {:primary true
+             :disabled submit-disabled?
+             :pending? @(subscribe [:ens.set-subnode-owner/tx-pending?
+                                    (namehash full-name)
+                                    (when subname (sha3 subname))])
+             :pending-text "Creating subdomain..."
+             :on-click (fn []
+                         (dispatch [:ens/set-subnode-owner @form-data]))}
+            "Create Subname"]]]]))))
+
+(defn transfer-ownership-form [defaults]
+  (let [form-data (r/atom (default-transfer-name-form-data defaults))]
+    (fn []
+      (let [{:keys [:ens.record/name :ens.record/owner]} @form-data
+            ownership-status @(subscribe [:ens.record/ownership-status (when (seq name)
+                                                        (str name constants/registrar-root))])
+            full-name (when (seq name)
+                        (str name constants/registrar-root))
+
+            label (name-label name)
+            submit-disabled? (not= ownership-status :ens.ownership-status/owner)
+            top-level? (top-level-name? full-name)
+            [transfer-event pending-sub] (if top-level?
+                                           [[:registrar/transfer {:ens.record/label name
+                                                                  :ens.record/owner owner}]
+                                            [:registrar.transfer/tx-pending? name]]
+                                           [[:ens/set-owner {:ens.record/name full-name
+                                                             :ens.record/owner owner}]
+                                            [:ens.set-owner/tx-pending? (namehash full-name)]])
+            node-hash (sha3 name)
+            registrar-entry @(subscribe [:registrar/entry node-hash])]
+        [ui/Grid
+         {:class "layout-grid submit-footer offering-form"}
+         [ui/GridRow
+          [ui/GridColumn
+           {:width 16}
+           [ui/Grid
+            {:relaxed "very"}
+            [ui/GridColumn
+             {:computer 8
+              :mobile 16}
+             [ens-name-input-ownership-validated
+              {:value name
+               :on-change #(swap! form-data assoc :ens.record/name (aget %2 "value"))}]]
+            [ui/GridColumn
+             {:computer 8
+              :mobile 16}
+             [address-text-field
+              {:value owner
+               :on-change #(swap! form-data assoc :ens.record/owner (aget %2 "value"))}]]]]]
+         [ui/GridRow
+          [ui/GridColumn
+           {:mobile 16
+            :class "join-upper"}
+           [:p.input-info
+            "By transferring ownership you're giving a new owner full control over ENS name as well as its locked funds"]
```

Comment:
Change this to "By transferring ownership you're giving full control over ENS name as well as its locked funds to a new owner."